ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/utils.py

```diff
@@ -585,3 +585,44 @@ def _local_path_groups_connected_to_pathfinder(self) -> list:
             for path_group in self.shared_utils.wan_local_path_groups
             if any(wan_interface["connected_to_pathfinder"] for wan_interface in path_group["interfaces"])
         ]
+
+    @cached_property
+    def _svi_acls(self) -> dict[str, dict[str, dict]]:
+        """
+        Returns a dict of
+            <interface_name>: {
+                "ipv4_acl_in": <generated_ipv4_acl>,
+                "ipv4_acl_out": <generated_ipv4_acl>,
+            }
+        Only contains interfaces with ACLs and only the ACLs that are set,
+        so use `get(self._svi_acls, f"{interface_name}.ipv4_acl_in")` to get the value.
+        """
+
+        svi_acls = {}
+        for tenant in self.shared_utils.filtered_tenants:
+            for vrf in tenant["vrfs"]:
+                for svi in vrf["svis"]:
+                    ipv4_acl_in = get(svi, "ipv4_acl_in")
+                    ipv4_acl_out = get(svi, "ipv4_acl_out")
+                    if ipv4_acl_in is None and ipv4_acl_out is None:
+                        continue
+
+                    interface_name = svi["name"]
```

Comment:
This is not correct for SVIs. The name is just a random name like "foo" and multiple VLANs can have the same name. Please use `f"Vlan{svi['id']}"` instead.

Reply:
Made the changes.